packages/lib/src/components/Giftcard/components/types.ts

```diff
@@ -0,0 +1,20 @@
+import Language from '../../../language/Language';
+
+export type GiftcardFieldsProps = {
+    setRootNode: (input: HTMLElement) => void;
+    i18n: Language;
+    pinRequired: boolean;
+    sfpState: any; //TODO
```

Comment:
If I am not mistaken, we have the type of this property defined as `SFPState` on `SFP/types.ts` file.

Shall we add the missing types here too?

Reply:
Fixed